sidecar/sokuji_sidecar/asr_engine.py

```
@@ -150,14 +169,298 @@ def flush(self):
+                continue
+            if data is None:
+                break
+            if self._mode == "always_stream":
+                await self._drive_always(send, data)
+            else:
+                await self._drive_utterance(send, data)
+        if self._mode == "always_stream":
+            if self._stream is not None and self._pending.strip():
+                try:
+                    final = await loop.run_in_executor(None, self._stream.end)
+                except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)